src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/AWS/SQS/CreateQueueAsyncIntegration.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Threading;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.SQS
+{
+    /// <summary>
+    /// AWSSDK.SQS CreateQueueAsync calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "AWSSDK.SQS",
+        TypeName = "Amazon.SQS.AmazonSQSClient",
+        MethodName = "CreateQueueAsync",
+        ReturnTypeName = "System.Threading.Tasks.Task`1<Amazon.SQS.Model.CreateQueueResponse>",
+        ParameterTypeNames = new[] { "Amazon.SQS.Model.CreateQueueRequest", ClrNames.CancellationToken },
+        MinimumVersion = "3.0.0",
+        MaximumVersion = "3.*.*",
+        IntegrationName = AwsConstants.IntegrationName)]
+    public class CreateQueueAsyncIntegration
+    {
+        private const string Operation = "CreateQueue";
+
+        /// <summary>
+        /// OnMethodBegin callback
+        /// </summary>
+        /// <typeparam name="TTarget">Type of the target</typeparam>
+        /// <typeparam name="TCreateQueueRequest">Type of the request object</typeparam>
+        /// <param name="instance">Instance value, aka `this` of the instrumented method</param>
+        /// <param name="request">The request for the SQS operation</param>
+        /// <param name="cancellationToken">CancellationToken value</param>
+        /// <returns>Calltarget state value</returns>
+        public static CallTargetState OnMethodBegin<TTarget, TCreateQueueRequest>(TTarget instance, TCreateQueueRequest request, CancellationToken cancellationToken)
+            where TCreateQueueRequest : ICreateQueueRequest, IDuckType
+        {
+            if (request.Instance is null)
+            {
+                return CallTargetState.GetDefault();
+            }
+
+            var scope = AwsSqsCommon.CreateScope(Tracer.Instance, $"{AwsConstants.AwsService}.{Operation}", out AwsSqsTags tags);
+            tags.Operation = Operation;
+            tags.Service = AwsConstants.AwsService;
```

Comment:
If service name is going to become `sqs` [as mentioned in this comment](https://github.com/DataDog/dd-trace-dotnet/pull/1471#issuecomment-844462806), is it still worth having this tag? I'm guessing so, given service name can be customised based on `DD_SERVICE`, just checking.

Reply:
My gut says yes, and I think it would also help to have `aws.service=sqs` as a tag for doing filtering in the UI as well